app/assets/stylesheets/forms.scss

```diff
@@ -0,0 +1,25 @@
+.user-forms {
+  border-radius: 10px;
+
+  label {
+    font-size: small;
+    margin-bottom: 0;
+  }
+
+  input {
+    font-size: small;
+  }
```

Comment:
Bootstrap has something for this

Reply:
I've been using fonts that are smaller that Bootstrap smallest font helper class for the Forms labels.
This feels wrong, but I've been trying to replicate the mock ups as accurately.

Reply:
Sorry I was wrong - there is the "small" class.